underlay/src/main/java/bio/terra/tanagra/filterbuilder/SchemaUtils.java

```diff
@@ -20,8 +20,7 @@ public static Literal toLiteral(Value value) {
       case TIMESTAMP_VALUE:
         return Literal.forTimestamp(
             new Timestamp(
-                value.getTimestampValue().getSeconds() * 1000
-                        + value.getTimestampValue().getNanos() * 1
+                value.getTimestampValue().getSeconds() * 1000 + value.getTimestampValue().getNanos()
                     ^ -6));
```

Comment:
This is not due to your changes here, but instead of this modification, I think we should replace this part with [below] for better readability.
```
        return Literal.forTimestamp(
                Timestamp.from(Instant
                        .ofEpochSecond(value.getTimestampValue().getSeconds() , value.getTimestampValue().getNanos())));
```

Reply:
done